mala/datahandling/ldos_aligner.py

```diff
@@ -279,30 +299,34 @@ def align_ldos_to_ref(
                 snapshot.output_npy_directory, save_path_ext
             )
             save_name = snapshot.output_npy_file
-
-            stripped_output_file_name = snapshot.output_npy_file.replace(
-                ".out", ""
-            )
-            ldos_shift_info_save_name = stripped_output_file_name.replace(
-                ".npy", ".ldos_shift.info.json"
-            )
-
+            target_name = os.path.join(save_path, save_name)
             os.makedirs(save_path, exist_ok=True)
 
-            if "*" in save_name:
-                save_name = save_name.replace("*", str(idx))
-                ldos_shift_info_save_name.replace("*", str(idx))
+            self.ldos_parameters.ldos_gridsize = ldos_shifted.shape[-1]
```

Comment:
This is needed for creation of correct dimensions in openPMD writing. The LDOS aligner apparently has some awareness of MPI. 
`left_index_ref` is computed by rank 0 and then distributed to others, `e_index_cut` however seems to be computed by everyone independently based on local data. I'm not sure if this can possibly yield consistent results, even with pure Numpy.
But in case this module actually supports MPI, then this is entirely untested for openPMD atm.

Reply:
For now, I would not worry about MPI, since this is experimental work for the most part, and the mass density project, in which this module is used, is still ongoing. Maybe we add a warning in the beginning that advises users not to trust results if MPI is used with the aligner?